details_trade.md

```diff
@@ -51,6 +51,20 @@ Livestock products were held fixed at their historical value for the rest of the
 ## No Trade of Secondary energy goods
 Note that secondary energy products such as [Electricity](energy.html#electricity) or [Refined Liquids](energy.html#refining) are assumed to not be traded at all between the GCAM geo-political regions.
 
```

Comment:
If needed, we can include links to trade parameters used in GCAM.
Descriptions of forest trade will need to be included for GCAM 5.4.